source/simplify.d.ts

```diff
@@ -1,3 +1,24 @@
+import {Class, Constructor} from './basic';
+
+/**
+Simplify options.
+
+@param deep - Do the simplification recursively (default: false).
```

Comment:
This should be attached to the `deep` property itself. And use `@default` for the default value